src/Stepper/Step.jsx

```diff
@@ -0,0 +1,340 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+import ReactDOM from 'react-dom';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /*
+      Customize the style of children wrapper div (include control button group and children)
+    */
+    childrenWrapperStyle: PropTypes.object,
+    /*
+      Customize the style of connector line
+    */
+    connectorLineStyle: PropTypes.object,
+     /*
+      Should be control buttons to handle how to move to next step
+    */
+    controlButtonsGroup: PropTypes.node,
+     /*
+     Customize the style of control button group wrapper
+    */
+    controlButtonsGroupWrapperStyle: PropTypes.object,
+     /*
+     the step is selected or not
+    */
+    isActive: PropTypes.bool,
+     /*
+     the step is completed or not
+    */
+    isCompleted: PropTypes.bool,
+     /*
+     the step is last step or not (which passed by automatically from Steps wrapper parent, no need
+     to pass directly)
+    */
+    isLastStep: PropTypes.bool,
+    /*
+      the header step is hovered or not
+    */
+    isStepHeaderHovered: PropTypes.bool,
+
+    /*
+      handle event when header step is hovered
+    */
+    onHeaderStepHover: PropTypes.func,
+    /*
+      handle when header step is touch
+    */
+    onHeaderStepTouch: PropTypes.func,
+    /*
+      the order of step which is shown in left avatar
+    */
+    orderStepLabel: PropTypes.string,
+    /*
+      the first optional step in previous steps
+    */
+    previousStepOptionalIndex: PropTypes.number,
+    /*
+      Customize style of step container which include connector line and children
+    */
+    stepContainerStyle: PropTypes.object,
+    /*
+      Customize of step header style
+    */
+    stepHeaderStyle: PropTypes.object,
+    /*
+     Customize of step header wrapper
+    */
+    stepHeaderWrapperStyle: PropTypes.object,
+    /*
+     The index of step in array of Steps
+    */
+    stepIndex: PropTypes.number,
+    /*
+      The label of step which shows in step header (default)
+    */
+    stepLabel: PropTypes.string,
+     /*
+      Customize step header view by pass DOM Node
+    */
+    stepLabelView: PropTypes.node,
+  },
+
+  contextTypes: {
+    muiTheme: PropTypes.object,
+    completedIcon: PropTypes.node,
```

Comment:
To be confirmed, but I think that React is going to allow only one property to be passed with the context. Can't we use a property here?